src/SFA.DAS.EmployerAccounts.Web/Views/Shared/_Layout.cshtml

```diff
@@ -44,6 +44,7 @@
     <!--[if IE 8]><link href="@Styles.Url("~/bundles/screenie8")" media="screen" rel="stylesheet" /><![endif]-->
     <!--[if gt IE 8]><!-->
     <link href="@Styles.Url("~/bundles/screen")" media="all" rel="stylesheet" />
+    <link href="@Html.CdnLink("css", "main.css")" rel="stylesheet" />

```

Comment:
cdn is still going to need to be removed as it overwrites a lot of the styling throughout the rest of the site, Front end are doing the work to migrate over the styles for the call to action